app/assets/stylesheets/shared/layout.scss

```diff
@@ -114,3 +114,51 @@ div.row.volunteer-filters {
     }
   }
 }
+
+
+
+@media only screen and (max-width: 2560px) {
+
+  #casa-cases_paginate.dataTables_paginate.paging_simple_numbers {
```

Comment:
What's the reason for using the id of the tables in these? CSS won't get propagated to all data tables.

Reply:
@schoork How do you propose to target the pagination? I am currently using CSS media queries to target different classes to make the pagination responsive. Can you guide me should I not use the CSS and use some JavaScript instead?

Reply:
I just wouldn't use an id here.
```
#casa-cases_paginate.dataTables_paginate.paging_simple_numbers {

.dataTables_paginate.paging_simple_numbers {
```